crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -93,8 +93,8 @@ def start_step(step_name)
           @logger.warn("The step #{step_name} doesn't exist")
           raise Crowbar::Error::StartStepExistenceError.new(step_name)
         end
-        if running? step_name
-          @logger.warn("The step has already been started")
+        if running?
+          @logger.warn("Some step is already running.")
           raise Crowbar::Error::StartStepRunningError.new(step_name)
```

Comment:
Hm, you are right, I should probably remove the argument from the exception completely